### PR TITLE
W-8946086

### PR DESCRIPTION
**Pull request by jacobisaliveandwell**
This commit changes the groupId and strips down the pom.xml to
bare essentials for our fork.

A strata config has been added for salesforce CI.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <version>7</version>
   </parent>
 
-  <groupId>com.clearspring.analytics</groupId>
+  <groupId>com.evergage</groupId>
```
Comment (ispringer) [inline pom.xml:10]: new package should be `com.evergage.thirdparty`

**Review by ispringer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jacobisaliveandwell) [reply pom.xml]: fixed!

**Review by ispringer — APPROVED** [review]

**Comment by jacobisaliveandwell** [comment]
@ispringer Do we want to update the version to be `-evg1`, or stick with `astral`? I have to update evergage-product either way.

**Comment by ispringer** [comment]
> @ispringer Do we want to update the version to be `-evg1`, or stick with `astral`? I have to update evergage-product either way.

Good idea. Let's change it to `evg1` for consistency with all the other forks.

**Review by ispringer — APPROVED** [review]

**Review by ispringer — APPROVED** [review]

**Pull request merged**